.github/workflows/build.yml

```diff
@@ -14,8 +14,41 @@ jobs:
           java-version: 11
       - name: Setup A+ LMS locally
         run: chmod +x start_local_a+_env.sh && ./start_local_a+_env.sh
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Cache Gradle packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
```

Comment:
Just curious, are these needed or just an improvement in how long the action takes to run?

Reply:
with cache: https://github.com/Aalto-LeTech/intellij-plugin/actions/runs/568557365
without cache: https://github.com/Aalto-LeTech/intellij-plugin/actions/runs/568555658

so caching is build is shorter (see the `build` jobs)